{{cookiecutter.folder_name}}/.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do we want to encourage/discourage the user from editing this after the initial cookiecutter checkout? If we want to encourage editing, we may name this `.config.json` or `.start_paper.cfg` or `.startrc` or something.